assets/scripts/src/choices.js

```diff
@@ -282,9 +281,9 @@ class Choices {
     // Reinstate passed element
     this.passedElement.classList.remove(this.config.classNames.input, this.config.classNames.hiddenState);
     this.passedElement.removeAttribute('tabindex');
-    this.passedElement.removeAttribute('style', 'display:none;');
+    this.passedElement.removeAttribute('style');
```

Comment:
It will. I automatically corrected this line, not thinking much. But your code `.setAttribute('style', 'display:none;')` erases all the styles previously set on the passed element. Need to investigate, thanks!